pypowsybl/network/impl/network.py

```diff
@@ -392,7 +392,7 @@ def write_network_area_diagram(self, svg_file: PathOrStr, voltage_level_ids: Uni
 
     def get_network_area_diagram(self, voltage_level_ids: Union[str, List[str]] = None, depth: int = 0,
                                  high_nominal_voltage_bound: float = -1, low_nominal_voltage_bound: float = -1,
-                                 nad_parameters: NadParameters = None) -> Svg:
+                                 nad_parameters: NadParameters = None, fixed_positions: str = None) -> Svg:
```

Comment:
It would be a lot more handy to have a dataframe of positions here if we want to process the positions in python. And then a utility method to get such a dataframe from the metadata json

Reply:
we must not forget that, currently, in the json metadata are  encoded the text nodes' positions , other than the nodes' positions (and could it be that, in the future, the json would contain some more graph elements?),  so 
a single dataframe parameter would not be enough in the get_network_area_diagram python api.  
Besides, the get_network_area_diagram alredy returns a couple: the svg and the json metadata.
I wonder if it could not be better to keep it simpler, leave here the json str metadata parameter and (in the future when it is needed) write a python utility/adapter method that creates a suitable json metadata from one or more positions dataframes (and vice versa a json -> multiple dataframes converter). What do you think?